diff --git a/src/graphics/niche/InkHUD/PlatformioConfig.ini b/src/graphics/niche/InkHUD/PlatformioConfig.ini
index 80984f399..a6d1a0fa2 100644
--- a/src/graphics/niche/InkHUD/PlatformioConfig.ini
+++ b/src/graphics/niche/InkHUD/PlatformioConfig.ini
@@ -1,7 +1,7 @@
 [inkhud]
-build_src_filter = 
+build_src_filter =
   +<graphics/niche/>; Include the nicheGraphics directory
-build_flags = 
+build_flags =
   -D MESHTASTIC_INCLUDE_NICHE_GRAPHICS ; Use NicheGraphics
   -D MESHTASTIC_INCLUDE_INKHUD ; Use InkHUD (a NicheGraphics UI)
   -D MESHTASTIC_EXCLUDE_SCREEN ; Suppress default Screen class
