diff --git a/platformio.ini b/platformio.ini
index 25997e11d..6d7315582 100644
--- a/platformio.ini
+++ b/platformio.ini
@@ -3,6 +3,8 @@
 
 [platformio]
 default_envs = tbeam
+boards_dir = rakwireless/boards
+; other required definitions
 
 extra_configs =
 	variants/*/*.ini
@@ -218,3 +220,26 @@ lib_deps =
 	sensirion/Sensirion Core@0.7.2
 	# renovate: datasource=custom.pio depName=Sensirion I2C SCD4x packageName=sensirion/library/Sensirion I2C SCD4x
 	sensirion/Sensirion I2C SCD4x@1.1.0
+
+; env:RAK4631_rak4631
+[env:rak4630]
+platform = nordicnrf52
+board = rak4630 
+framework = arduino
+build_src_filter = ${env.build_src_filter}+<../rakwireless/variants/rak4630>
+build_flags = -I rakwireless/variants/rak4630
+; other build flags
+lib_deps = adafruit/Adafruit LIS3DH
+; project library dependencies
+; other required environment definitions
+
+[env:rak4631]
+platform = nordicnrf52@^10.8.0
+board = rak4631
+framework = arduino
+lib_deps = 
+    adafruit/Adafruit LIS3DH
+build_flags = 
+    -D RAK4631
+    -D USE_LIS3DH_SENSOR
+    -I variants/rak4631
