diff --git a/src/mesh/generated/meshtastic/telemetry.pb.cpp b/src/mesh/generated/meshtastic/telemetry.pb.cpp
index 345d7a157..944689104 100644
--- a/src/mesh/generated/meshtastic/telemetry.pb.cpp
+++ b/src/mesh/generated/meshtastic/telemetry.pb.cpp
@@ -27,6 +27,9 @@ PB_BIND(meshtastic_HealthMetrics, meshtastic_HealthMetrics, AUTO)
 PB_BIND(meshtastic_HostMetrics, meshtastic_HostMetrics, 2)
 
 
+PB_BIND(meshtastic_TelemetryMotion, meshtastic_TelemetryMotion, AUTO)
+
+
 PB_BIND(meshtastic_Telemetry, meshtastic_Telemetry, 2)
 
 
