diff --git a/src/modules/SeismicModule.h b/src/modules/SeismicModule.h
new file mode 100644
index 000000000..13f919b52
--- /dev/null
+++ b/src/modules/SeismicModule.h
@@ -0,0 +1,12 @@
+#pragma once
+#include "../concurrency/OSThread.h"
+
+class SeismicModule : public concurrency::OSThread {
+public:
+    SeismicModule();
+    virtual int32_t runOnce() override;
+private:
+    bool initLIS3DH();
+    int16_t read16(uint8_t reg);
+    void write8(uint8_t reg, uint8_t v);
+};
